src/backend/distributed/metadata/dependency.c

```diff
@@ -776,15 +780,11 @@ DeferErrorIfHasUnsupportedDependency(const ObjectAddress *objectAddress)
 	#endif
 
 	/*
-	 * If the given object is a procedure or type, we want to create it locally,
-	 * so provide that information in the error detail.
+	 * Then we want to create it locally, so provide that information
```

Comment:
Let's update this comment by referencing the change on the dependencies.c file. We will use that detail if we are giving a warning but not use it while erroring out.